docs/variant_annotation.md

```diff
@@ -160,9 +160,13 @@ hence are not present in the final BigQuery output table._
 ### Querying annotations directly
 
 We imported the gnomAD genomes table using the features described in
-[Parsing Annotation Fields](#parsing-annotation-fields). Here is a sample
-query to extract high impact variants in the HBB gene which encodes the beta
-chain of Haemoglobin:
+[Parsing Annotation Fields](#parsing-annotation-fields).
+
+The [gnomAD sample query notebook](sample_queries) has additional queries
```

Comment:
Could you please also include a link from our main README file to the sample queries page to make it more visible?

Reply:
Done below